tedana/selection/select_comps.py

```diff
@@ -25,9 +25,8 @@ def selcomps(seldict, comptable, mmix, manacc, n_echos):
     seldict : :obj:`dict`
         A dictionary with component-specific features used for classification.
         As output from `fitmodels_direct`
-    comptable : (C x X) :obj:`pandas.DataFrame`
-        Component metric table. One row for each component, with a column for
-        each metric. The index should be the component number.
+    comptable : (C x 5) :obj:`pandas.DataFrame`
+        Component metric table
```

Comment:
Should the docstring explain the shape, still ?

Reply:
I'm not sure how this was changed. Reverting it back now.